modules/custom/external_comment/external_comment.module

```diff
@@ -25,3 +25,78 @@ function external_comment_query_comment_filter_alter(AlterableInterface $query)
     }
   }
 }
+
+
+
+function external_comment_cron() {
+  // fetch comments
+  $comment_ids = \Drupal::entityQuery('comment')
```

Comment:
you may want to start with nodes?

Reply:
I don't fully understand this, but let me know if it's still applicable after the new changes.